Source/ParameterEncoding.swift

```diff
@@ -87,6 +87,8 @@ public struct URLEncoding: ParameterEncoding {
         case noBrackets
         /// Brackets containing the item index are appended. This matches the jQuery and Node.js behavior.
         case indexInBrackets
+        /// Provide a custom array key encoding with the given closure.
+        case custom((String, Int) -> String)
```

Comment:
```suggestion
        case custom((_ key: String, _ index: Int) -> String)
```

Internal labels make this a bit nicer to read.